public/stylesheets/style.css

```diff
@@ -473,6 +479,11 @@ body>header img {
   border-radius: .8rem;
   color: white;
 }
+
+  #alphatical.home {
+       margin-top: 12em;
+    }
```

Comment:
Let even op je indentation. Hier zijn de spaties een beetje overhoop en eerder al gebruik je zo te zien een tab in plaats van 2 spaces